src/api/bereikbaarheid/index.ts

```diff
@@ -3,9 +3,10 @@
 //
 
 import axios from 'axios'
+import config from 'config'
 
 export const api = axios.create({
-  baseURL: process.env.REACT_APP_API_ROOT,
+  baseURL: process.env.NODE_ENV === 'test' ? process.env.REACT_APP_API_ROOT : config.API_ROOT,
```

Comment:
Is it not cleaner to do all this logic in the config file?

Reply:
All logic is now in the config file. Some rigorous refactoring to the test files was needed.